### PR TITLE
Update okrs.md

### PR DESCRIPTION
**Pull request by TH33765**
## Why
* These revisions add further process around the reporting of quarterly OKRs, starting in Q2, 2024 to provide the Level 1 Objective owner deeper and more timely line of sight into efforts of KR owners to move the needle.

## What
* Additional requirements for weekly, and quarterly, reporting and documentation has been added to our OKR page in the OS.

### CODE REVIEW

File: docs/process/okrs.md [hunk 1]
```diff
@@ -23,3 +23,10 @@ When establishing OKRs, we adopt a scientific approach to avoid setting unattain
...
 
 We have embraced OKRs to drive our mission of refactoring healthcare. We adopted OKRs early on, recognizing the importance of everyone understanding our top priorities and how to measure our impact. We set OKRs at the company and team levels, consistently reviewing our progress and learning from challenges. OKRs serve as a catalyst for meaningful conversations, valuable feedback, and recognition within our teams, enhancing both performance and our company culture. We firmly believe that OKRs, along with regular progress tracking and check-ins, are key factors in our success, and we constantly strive to leverage them even better.
+
+### Each quarter, there is one owner of each Level 2 Objective and accompanying KRs. They are responsible for moving the needle on their Objective and accompanying KRs, and they report on a weekly basis to the Level 1 Objective owner through 2 mechanisms.
```
Comment (jdjkelly) [inline docs/process/okrs.md:27]:
```suggestion
### Each quarter, there is one owner of each Level 2 Objective and accompanying KRs. They are responsible for moving the needle on their Objective and accompanying KRs, and they report on a weekly basis to the Level 1 Objective owner through 2 mechanisms.
```

This should be a level heading like `## Reporting` and then this should be first paragraph.

I think also that this document should outline what "Level 2 Objectives" are - the idea is first introduced here, but isn't explained.

**Review by jdjkelly — CHANGES_REQUESTED** [review]

File: docs/process/okrs.md (hunk 1 above)
Comment (amjarruda) [reply docs/process/okrs.md:27]: Good comments. Good progress.

**Review by TH33765 — COMMENTED** [review]
further edits and refinement of original draft based on Josh's feedback.

**Comment by TH33765** [comment]
@angelaliu22 just finalized my edits as last quarter's company level O owner --> baton now over to you for this quarter if you want to edit anything for the OS

**Comment by TH33765** [comment]
@jdjkelly I'm trying to ship the version of the OS edits around OKRs you and I discussed. It looks like you have a "changes requested" still pending. Is there anything I need to do on my end to resolve this?

Screenshot 2024-07-08 at 2 21 41 PM

**Review by jdjkelly — APPROVED** [review]

**Comment by jdjkelly** [comment]
@TH33765 you should be good to go now

**Pull request merged**